### PR TITLE
Add SEO cards , meta tags for Twitter, Facebook ,LinkedIn #336

### PR DESCRIPTION
**Pull request by dev-george-nikolaidis**
@gbowne1 

## 🛠 Changes
This PR boosts our website's SEO and social media presence with the addition of SEO cards and specific meta tags for Twitter, Facebook, and LinkedIn. Here's a quick rundown:

- **SEO Optimization**: Enhanced search engine visibility.
- **Twitter Cards**: Optimized content display on Twitter with tailored meta tags.
- **Facebook Open Graph**: Enriched Facebook shares with specific tags for better engagement.
- **LinkedIn Tags**: Improved link appearance on LinkedIn for professional outreach.

These updates aim to enhance our site's shareability and attractiveness on social platforms.
facebook-card
linkedin-card
x-card

How to test it
You can install the browser extension Social Share Preview to test it locally.

### CODE REVIEW

**Review by gbowne1 — APPROVED** [review]
- [x] checked out this change locally for testing and review for merge 

This PR adds the Twitter and OpenGraph SEO to ReactSocialNetwork.

I tested this locally and it seems to work as expected and also as shown here in the PR, although I know very little about the Twitter and OpenGraph stuff.

Approving this PR for merge, awaiting for reviews from our collaborators and maintainers @balajik @manuel12 @pawel975 reviews before merging.

Thanks for the opportunity to review your PR @dev-george-nikolaidis and for your contributions to this project

**Review by pawel975 — APPROVED** [review]

**Review by pawel975 — APPROVED** [review]

**Comment by gbowne1** [comment]
@balajik @manuel12 can you please review these? thanks

**Comment by gbowne1** [comment]
Merging this as complete and tested locally.

**Pull request merged**
